### PR TITLE
TERRA_ril GBS pipeline.Rmd

### PR DESCRIPTION
**Pull request by zhenbinHU**

### CODE REVIEW

File: genomics/01-bap_resequencing.Rmd [hunk 1]
```diff
@@ -14,11 +14,57 @@ output:
...
+                "Row", "Col", "LibraryPrepID", "Enzyme")
+write.table(key, "/home/rstudio/tutorials/genomics/key_v5",
+            quote = FALSE, row.names = FALSE, sep = "\t")
+```
 
+# index genome
+```{bash}
+cd tutorials
+mkdir ref_genome
+git clone https://github.com/lh3/bwa.git
```
Comment (dlebauer) [inline genomics/01-bap_resequencing.Rmd]: could this be replaced by `apt-get install bwa`?

**Comment by dlebauer** [comment]
What are the hardware requirements? How much RAM is needed?

File: genomics/01-bap_resequencing.Rmd (hunk 1 above)
Comment (zhenbinHU) [reply genomics/01-bap_resequencing.Rmd]: I don't think this can be replaced by "apt-get install bwa".  BWA is a bioinformatics package for alignment.

Comment (phloz) [reply genomics/01-bap_resequencing.Rmd]: @dlebauer @zhenbinHU look at the following. I'd structure it similarly...
 https://github.com/terraref/documentation/blob/master/genomic-data-product.md

**Comment by dlebauer** [comment]
@zhenbinHU could you update this pull request with the requested information?

**Comment by dlebauer** [comment]
closing orphaned effort

**Pull request closed without merging**